crates/ruff/src/checkers/ast/mod.rs

```diff
@@ -1288,17 +1288,6 @@ where
         self.semantic.flags = flags_snapshot;
     }
 
-    fn visit_format_spec(&mut self, format_spec: &'b Expr) {
```

Comment:
I'm assuming this visit is done elsewhere and that's why it's removed here?

Reply:
yes, this is now handled as part of walking f-strings (which walks the parts, some of which are formatted-values which has the format spec

see e.g. https://github.com/davidszotten/ruff/blob/0eafbe4afee2f43cfc1af3b6ccdec8479133d38b/crates/ruff_python_ast/src/visitor.rs#L705